src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -62,101 +62,118 @@ public static partial class AppHostBuilderExtensions
-			handlersCollection.AddHandler<GraphicsView, GraphicsViewHandler>();
-			handlersCollection.AddHandler<Image, ImageHandler>();
-			handlersCollection.AddHandler<Label, LabelHandler>();
-			handlersCollection.AddHandler<Layout, LayoutHandler>();
-			handlersCollection.AddHandler<Picker, PickerHandler>();
-			handlersCollection.AddHandler<ProgressBar, ProgressBarHandler>();
-			handlersCollection.AddHandler<ScrollView, ScrollViewHandler>();
-			handlersCollection.AddHandler<SearchBar, SearchBarHandler>();
-			handlersCollection.AddHandler<Slider, SliderHandler>();
-			handlersCollection.AddHandler<Stepper, StepperHandler>();
-			handlersCollection.AddHandler<Switch, SwitchHandler>();
-			handlersCollection.AddHandler<TimePicker, TimePickerHandler>();
-			handlersCollection.AddHandler<Page, PageHandler>();
-			handlersCollection.AddHandler<WebView, WebViewHandler>();
-			handlersCollection.AddHandler<Border, BorderHandler>();
-			handlersCollection.AddHandler<IContentView, ContentViewHandler>();
-			handlersCollection.AddHandler<Shapes.Ellipse, ShapeViewHandler>();
-			handlersCollection.AddHandler<Shapes.Line, LineHandler>();
-			handlersCollection.AddHandler<Shapes.Path, PathHandler>();
-			handlersCollection.AddHandler<Shapes.Polygon, PolygonHandler>();
-			handlersCollection.AddHandler<Shapes.Polyline, PolylineHandler>();
-			handlersCollection.AddHandler<Shapes.Rectangle, RectangleHandler>();
-			handlersCollection.AddHandler<Shapes.RoundRectangle, RoundRectangleHandler>();
-			handlersCollection.AddHandler<Window, WindowHandler>();
-			handlersCollection.AddHandler<ImageButton, ImageButtonHandler>();
-			handlersCollection.AddHandler<IndicatorView, IndicatorViewHandler>();
-			handlersCollection.AddHandler<RadioButton, RadioButtonHandler>();
-			handlersCollection.AddHandler<RefreshView, RefreshViewHandler>();
-			handlersCollection.AddHandler<SwipeItem, SwipeItemMenuItemHandler>();
-			handlersCollection.AddHandler<SwipeView, SwipeViewHandler>();
+			handlersCollection.AddHandler<CollectionView>(_ => new CollectionViewHandler());
+			handlersCollection.AddHandler<CarouselView>(_ => new CarouselViewHandler());
+			handlersCollection.AddHandler<Application>(_ => new ApplicationHandler());
+			handlersCollection.AddHandler<ActivityIndicator>(_ => new ActivityIndicatorHandler());
+			handlersCollection.AddHandler<BoxView>(_ => new BoxViewHandler());
+			handlersCollection.AddHandler<Button>(_ => new ButtonHandler());
+			handlersCollection.AddHandler<CheckBox>(_ => new CheckBoxHandler());
+			handlersCollection.AddHandler<DatePicker>(_ => new DatePickerHandler());
+			handlersCollection.AddHandler<Editor>(_ => new EditorHandler());
+			handlersCollection.AddHandler<Entry>(_ => new EntryHandler());
+			handlersCollection.AddHandler<GraphicsView>(_ => new GraphicsViewHandler());
+			handlersCollection.AddHandler<Image>(_ => new ImageHandler());
+			handlersCollection.AddHandler<Label>(_ => new LabelHandler());
```

Comment:
If I put this line back, it fixes Tim's app: https://github.com/dotnet/maui/issues/17028

Makes no sense, feels like we are using Microsoft.Extensions.DI wrong?